arrow/src/test/java/org/apache/iceberg/arrow/vectorized/ArrowReaderTest.java

```diff
@@ -262,6 +264,111 @@ public void testReadColumnFilter2() throws Exception {
         scan, NUM_ROWS_PER_MONTH, 12 * NUM_ROWS_PER_MONTH, ImmutableList.of("timestamp"));
   }
 
+  @Test
+  public void testReadColumnThatDoesNotExistInParquetSchema() throws Exception {
+    setMaxStackTraceElementsDisplayed(15);
```

Comment:
do we really need this?

Reply:
This tests the exact scenario that is the impetus for adding support for null vectors. I feel this test is essential. This test is the only test exercising this pull request. If you see things differently, please explain.

Reply:
@sl255051 why is this needed?

Reply:
The assertion stack trace is abridged before it is written out. I added that line because at that time the abridged stack trace didn't show the method of interest. Calling `setMaxStackTraceElementsDisplayed(15)` changes how the stack trace is abridged. But I don't think this is needed now that the code has changed with your help. I have removed this line.